src/Apps/Sell/SellFlowContext.tsx

```diff
@@ -97,10 +101,17 @@ export const SellFlowContextProvider: React.FC<SellFlowContextProviderProps> = (
     push(`/sell2/submissions/${submissionID}/${STEPS[index]}`)
   }, [push, submissionID, index])
 
+  const getSubmissionState = () => {
+    return state.isSubmitStep ? "SUBMITTED" : "DRAFT"
+  }
```

Comment:
Sorry for sneaking this update into this PR. We want to set the state of the submission depending on the step.

Reply:
It feels a bit excessive to pass a `state` value for every create/update mutation request. This might also result in submissions fliping from submitted > draft > submission if a user goes back via the URL to update submission info.

I wonder if we could instead update the `finishFlow` function to fire a second updateSubmission mutation with just `state: SUBMITTED`. This will likely become conditional in future iterations as we hope to enable collectors to update information in their submission without necessarily re-submitting.

Reply:
Good catch! Probably not a good idea to set the state to "DRAFT" again when going back after submitting. I've updated the context so the state is only set when clicking "Submit" in the last step. This works because it looks like `step` defaults to "DRAFT" if it's not set explicitly.

Reply:
https://github.com/artsy/force/pull/14041/commits/a76cdef72baff5d65e811fb0cf92ffd03b1f08bf

Reply:
Nice 👍🏼